scripts/zones/Boneyard_Gully/bcnms/like_the_wind.lua

```diff
@@ -35,7 +35,8 @@ end;
 function onBcnmLeave(player,instance,leavecode)
 
     if (leavecode == 2) then -- play end CS. Need time and battle id for record keeping + storage
-        player:startEvent(32001,1,1,1,instance:getTimeInside(),1,0,0);
+        player:startEvent(32002,1,1,1,instance:getTimeInside(),1,0,0);
+        player:setVar("ENM_Win",1)
```

Comment:
I'm 9000% sure we shouldn't be doing this with that variable. leavecode should already be indicating if we won or not.